src/client/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -199,6 +199,7 @@ public void LogParameters()
             builder.AppendLine("CorrelationId - " + CorrelationId);
             builder.AppendLine("UserAssertion set: " + (UserAssertion != null));
             builder.AppendLine("LongRunningOboCacheKey set: " + !string.IsNullOrWhiteSpace(LongRunningOboCacheKey));
+            builder.AppendLine("Region configured: " + AppConfig.AzureRegion);
```

Comment:
I think with the configured region we also need the source of region discovery. If we are suspecting the region discovered is wrong then it could be either environment variable or imds

Reply:
Agreed, but that is actually logged as part of the response.